src/emailservice/email_server.rb

```diff
@@ -0,0 +1,22 @@
+require "ostruct"
+require "pony"
+require "sinatra"
+
+require "opentelemetry/sdk"
+require "opentelemetry/exporter/otlp"
+require "opentelemetry/instrumentation/sinatra"
```

Comment:
Can we use Bundler here to load dependencies and use `Bundler.require` instead of requiring each dependency separately?

Reply:
I believe so, yes.